src/SlackMessages.ts

```diff
@@ -1,13 +1,11 @@
-import { Presentable } from "@atomist/rug/operations/Handlers";
-
 /**
  * Construct and render slack messages according to Slack message
  * formatting: https://api.slack.com/docs/message-formatting. Customize
  * messages with rug actions.
  */
 
 export function emptyString(str: string) {
-    return !str || str === "" || str === undefined;
+    return str == null || str === "";
```

Comment:
Yes, that's  how javascript interprets `undefined`. `blah != null` will guard against either `null` or `undefined` (basically `null == undefined` but `null !== undefined`...)